src/shared/utils/index.ts

```diff
@@ -90,19 +91,20 @@ function blobToValidators(blob: UNLBlob): Set<string> {
  *
  * @returns An array of sets containing the signing keys of validators on each list.
  */
-export async function getLists(): Promise<{
-  vl_main: Set<string>
-  vl_test: Set<string>
-  vl_dev: Set<string>
-}> {
-  const main_blob = await fetchValidatorList(config.vl_main)
-  const test_blob = await fetchValidatorList(config.vl_test)
-  const dev_blob = await fetchValidatorList(config.vl_dev)
-  return {
-    vl_main: blobToValidators(main_blob),
-    vl_test: blobToValidators(test_blob),
-    vl_dev: blobToValidators(dev_blob),
-  }
+export async function getLists(): Promise<Record<string, Set<string>>> {
+  const lists = {}
+  const promises: Array<Promise<void>> = []
+  networks.forEach(async (network) => {
+    promises.push(
+      fetchValidatorList(network.unls[0]).then((blob) => {
```

Comment:
Do we need to make any changes here, because the old values are `vl_main`, and the new network values are `main`?

Reply:
I just double-checked, but the only place where that's used was also refactored.